<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>909</width>
     <height>22</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen_project">
     <property name="title">
      <string>Open project</string>
     </property>
     <property name="icon">
      <iconset resource="../ressources.qrc">
       <normaloff>:/icons/Ressources/open_file.png</normaloff>:/icons/Ressources/open_file.png</iconset>
     </property>
     <addaction name="actionBrowse"/>
    </widget>
    <addaction name="actionNew_project"/>
    <addaction name="menuOpen_project"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_all"/>
    <addaction name="actionExport_standalone"/>
    <addaction name="actionClose_project"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Test</string>
    </property>
    <addaction name="actionPlay"/>
   </widget>
   <widget class="QMenu" name="menuManagement">
    <property name="title">
     <string>Management</string>
    </property>
    <addaction name="actionDatas_manager"/>
    <addaction name="actionSystems_manager"/>
    <addaction name="actionVariables_manager"/>
    <addaction name="actionCollisions_manager"/>
    <addaction name="actionKeyboard_controls"/>
    <addaction name="actionScripts_manager"/>
    <addaction name="actionPictures_manager"/>
    <addaction name="actionSongs_manager"/>
    <addaction name="actionShapes_manager"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionSet_BR_path_folder"/>
    <addaction name="actionDebug_options"/>
    <addaction name="actionGeneral_options"/>
    <addaction name="actionChange_language"/>
    <addaction name="actionAuto_update"/>
   </widget>
   <widget class="QMenu" name="menuSpecials">
    <property name="title">
     <string>Special elements</string>
    </property>
    <addaction name="actionAutotiles"/>
    <addaction name="actionAnimated_Autotiles"/>
    <addaction name="actionSprite_walls"/>
    <addaction name="action3D_objects"/>
    <addaction name="actionMountains"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Display</string>
    </property>
    <addaction name="actionShow_Hide_grid"/>
    <addaction name="actionShow_Hide_square_informations"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionHeight_up"/>
    <addaction name="actionHeight_plus_up"/>
    <addaction name="actionHeight_down"/>
    <addaction name="actionHeight_plus_down"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdition"/>
   <addaction name="menuManagement"/>
   <addaction name="menuSpecials"/>
   <addaction name="menuOptions"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuTest"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_project"/>
   <addaction name="actionBrowse"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_all"/>
   <addaction name="separator"/>
   <addaction name="actionDatas_manager"/>
   <addaction name="actionSystems_manager"/>
   <addaction name="actionVariables_manager"/>
   <addaction name="actionCollisions_manager"/>
   <addaction name="actionKeyboard_controls"/>
   <addaction name="actionScripts_manager"/>
   <addaction name="separator"/>
   <addaction name="actionPictures_manager"/>
   <addaction name="actionVideos_manager"/>
   <addaction name="actionSongs_manager"/>
   <addaction name="actionShapes_manager"/>
   <addaction name="separator"/>
   <addaction name="actionAutotiles"/>
   <addaction name="actionAnimated_Autotiles"/>
   <addaction name="actionSprite_walls"/>
   <addaction name="action3D_objects"/>
   <addaction name="actionMountains"/>
   <addaction name="separator"/>
   <addaction name="actionPlay"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit the engine</string>
   </property>
   <property name="statusTip">
    <string>Quit the engine</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNew_project">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/new_file.png</normaloff>:/icons/Ressources/new_file.png</iconset>
   </property>
   <property name="text">
    <string>New project...</string>
   </property>
   <property name="iconText">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionBrowse">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/open_file.png</normaloff>:/icons/Ressources/open_file.png</iconset>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
   <property name="iconText">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open an existing project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose_project">
   <property name="text">
    <string>Close project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/save.png</normaloff>:/icons/Ressources/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/play.png</normaloff>:/icons/Ressources/play.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Play your game!</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionDatas_manager">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/datas.png</normaloff>:/icons/Ressources/datas.png</iconset>
   </property>
   <property name="text">
    <string>Datas manager</string>
   </property>
   <property name="iconText">
    <string>Datas</string>
   </property>
   <property name="toolTip">
    <string>Datas manager of your project</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="actionVariables_manager">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/variables.png</normaloff>:/icons/Ressources/variables.png</iconset>
   </property>
   <property name="text">
    <string>Variables manager</string>
   </property>
   <property name="iconText">
    <string>Variables</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="actionSystems_manager">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/systems.png</normaloff>:/icons/Ressources/systems.png</iconset>
   </property>
   <property name="text">
    <string>Systems manager</string>
   </property>
   <property name="iconText">
    <string>Systems</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionKeyboard_controls">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/inputs.png</normaloff>:/icons/Ressources/inputs.png</iconset>
   </property>
   <property name="text">
    <string>Keyboard controls</string>
   </property>
   <property name="iconText">
    <string>Keyboard</string>
   </property>
   <property name="shortcut">
    <string>5</string>
   </property>
  </action>
  <action name="actionSave_all">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/save.png</normaloff>:/icons/Ressources/save.png</iconset>
   </property>
   <property name="text">
    <string>Save all</string>
   </property>
   <property name="iconText">
    <string>All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExport_standalone">
   <property name="text">
    <string>Export standalone...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionPictures_manager">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/pictures.png</normaloff>:/icons/Ressources/pictures.png</iconset>
   </property>
   <property name="text">
    <string>Pictures manager</string>
   </property>
   <property name="iconText">
    <string>Pictures</string>
   </property>
   <property name="shortcut">
    <string>8</string>
   </property>
  </action>
  <action name="actionSet_BR_path_folder">
   <property name="text">
    <string>Set BR path folder...</string>
   </property>
   <property name="shortcut">
    <string>Shift+U</string>
   </property>
  </action>
  <action name="actionAutotiles">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/autotile_toolbar.png</normaloff>:/icons/Ressources/autotile_toolbar.png</iconset>
   </property>
   <property name="text">
    <string>Autotiles</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionSprite_walls">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/walls_toolbar.png</normaloff>:/icons/Ressources/walls_toolbar.png</iconset>
   </property>
   <property name="text">
    <string>Sprite walls</string>
   </property>
   <property name="iconText">
    <string>Walls</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="action3D_objects">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/object_toolbar.png</normaloff>:/icons/Ressources/object_toolbar.png</iconset>
   </property>
   <property name="text">
    <string>3D objects</string>
   </property>
   <property name="iconText">
    <string>3D objects</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="actionMountains">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/mountain_toolbar.png</normaloff>:/icons/Ressources/mountain_toolbar.png</iconset>
   </property>
   <property name="text">
    <string>Mountains</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+5</string>
   </property>
  </action>
  <action name="actionShow_Hide_grid">
   <property name="text">
    <string>Show / Hide grid</string>
   </property>
   <property name="shortcut">
    <string>G</string>
   </property>
  </action>
  <action name="actionShow_Hide_square_informations">
   <property name="text">
    <string>Show / Hide square informations</string>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionAnimated_Autotiles">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/autotile_animated_toolbar.png</normaloff>:/icons/Ressources/autotile_animated_toolbar.png</iconset>
   </property>
   <property name="text">
    <string>Animated Autotiles</string>
   </property>
   <property name="iconText">
    <string>! Autotiles</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionSongs_manager">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/songs.png</normaloff>:/icons/Ressources/songs.png</iconset>
   </property>
   <property name="text">
    <string>Songs manager</string>
   </property>
   <property name="iconText">
    <string>Songs</string>
   </property>
   <property name="shortcut">
    <string>9</string>
   </property>
  </action>
  <action name="actionCollisions_manager">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/collisions.png</normaloff>:/icons/Ressources/collisions.png</iconset>
   </property>
   <property name="text">
    <string>Collisions manager</string>
   </property>
   <property name="iconText">
    <string>Collisions</string>
   </property>
   <property name="shortcut">
    <string>4</string>
   </property>
  </action>
  <action name="actionScripts_manager">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/scripts.png</normaloff>:/icons/Ressources/scripts.png</iconset>
   </property>
   <property name="text">
    <string>Scripts manager</string>
   </property>
   <property name="iconText">
    <string>Scripts</string>
   </property>
   <property name="shortcut">
    <string>6</string>
   </property>
  </action>
  <action name="actionDebug_options">
   <property name="text">
    <string>Debug options...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionGeneral_options">
   <property name="text">
    <string>General options...</string>
   </property>
   <property name="shortcut">
    <string>U</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHeight_up">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/height.png</normaloff>:/icons/Ressources/height.png</iconset>
   </property>
   <property name="text">
    <string>Height up</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Up</string>
   </property>
  </action>
  <action name="actionHeight_plus_up">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/height_plus.png</normaloff>:/icons/Ressources/height_plus.png</iconset>
   </property>
   <property name="text">
    <string>Height plus up</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Up</string>
   </property>
  </action>
  <action name="actionHeight_down">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/height_down.png</normaloff>:/icons/Ressources/height_down.png</iconset>
   </property>
   <property name="text">
    <string>Height down</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Down</string>
   </property>
  </action>
  <action name="actionHeight_plus_down">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/height_down_plus.png</normaloff>:/icons/Ressources/height_down_plus.png</iconset>
   </property>
   <property name="text">
    <string>Height plus down</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Down</string>
   </property>
  </action>
  <action name="actionShapes_manager">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/object_toolbar.png</normaloff>:/icons/Ressources/object_toolbar.png</iconset>
   </property>
   <property name="text">
    <string>Shapes manager</string>
   </property>
   <property name="iconText">
    <string>Shapes</string>
   </property>
   <property name="shortcut">
    <string>0</string>
   </property>
  </action>
  <action name="actionVideos_manager">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/play_song.png</normaloff>:/icons/Ressources/play_song.png</iconset>
   </property>
   <property name="text">
    <string>Videos manager</string>
   </property>
   <property name="iconText">
    <string>Videos</string>
   </property>
  </action>
  <action name="actionAuto_update">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto display updater</string>
   </property>
  </action>
  <action name="actionChange_language">
   <property name="text">
    <string>Change language...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
